### DISCUSSION TITLE
Feature Request: graph editor

### DISCUSSION THREAD

**Discussion by ArtistBirdy** [keiwando/toonsquid#98] [Ideas]
Please change this curve preset editor to a manual graph editor with bazier, nodes and handles like other keframe animation software. it’ll offer more flexibility for editing curves.

![96477D1A-8DDF-4025-B51D-C7AD94CE73CF](https://user-images.githubusercontent.com/118920457/208087710-93974f39-749c-4476-844b-b76b94be5029.png)

**Comment by ArtistBirdy** [comment]
Node video editor and Alight motion have this kind of anual curve editors

**Comment by Lioncub901** [comment]
I think you should add two modes. One for the presets and the other for customizability because I like the presets because I am lazy and because I mostly do traditional.
